views.py

```diff
@@ -400,3 +400,31 @@ def add_article(request):
     return render(request, template, context)
 
 
+@has_journal
+@editor_user_required
+def discount_apc(request, apc_id):
+    """
+    Allows editor to discount an APC.
+    """
+    apc = get_object_or_404(
+        models.ArticleAPC,
+        pk=apc_id,
+        article__journal=request.journal,
+    )
+
+    if request.POST:
+        new_apc_amount = request.POST.get('new_value')
+        apc.value = new_apc_amount
+        apc.save()
+        messages.add_message(
+            request,
+            messages.SUCCESS,
+            'APC Value Updated',
```

Comment:
Since you know the previous and new values, maybe we can add them to the message:
"APC value updated from {old_value} to {new_value}"